app/models/custom_form.py

```diff
@@ -1,6 +1,19 @@
 import json
+import binascii
+import os
 
 from app.models import db
+from app.api.helpers.db import get_count
+
+
+def get_new_custom_form_identifier(length=8):
```

Comment:
No no... this is not autogenerated. This `identifier` is to be set by the client. Better call this as `field_identifier` to avoid confusion